_data/companies.yml

```diff
@@ -2616,13 +2602,6 @@
   visitors: "?"
   events: Restricted
   last_update: 2020-03-10
-  
-- name: Vouch
-  wfh: Required
-  travel: Suspended
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
```

Comment:
Is this intentional?

Reply:
No idea why I only just now got this but, yeah, apologies all over. The step with the checkboexes threw me for a loop (don't github much) and I am pretty sure that I borked it then had to redo (at least once).

So very sorry.